scripts/hab2_benchmark.py

```diff
@@ -0,0 +1,227 @@
+import argparse
```

Comment:
benchmark is maybe a bit vague. `hab2_performance_benchmark` would be better and we could have rl / il benchmarks in the future